### PR TITLE
Added mobile support for Trac.

### PR DESCRIPTION
**Pull request by sarahboyce**

### CODE REVIEW

File: trac-env/templates/site_head.html [hunk 1]
```diff
@@ -1,3 +1,4 @@
+<meta name="viewport" content="width=device-width, initial-scale=1" />
```
Comment (bmispelon) [inline trac-env/templates/site_head.html:1]: This was the line I thought I could merge, not the rest of the CSS (which I think we can improve). Do you think it makes sense to merge this `<meta>` tag on its own, or does it only make sense in conjunction with the other CSS changes?

Comment (sarahboyce) [reply trac-env/templates/site_head.html:1]: It is possible to merge without the CSS, but I think the CSS changes improved the mobile experience quite a bit - especially the scrolling.
Which parts of the CSS do you want to improve?

Comment (sarahboyce) [reply trac-env/templates/site_head.html:1]: Pulled out: https://github.com/django/code.djangoproject.com/pull/214

**Comment by bmispelon** [comment]
This could be closed in favor of #142, no?

From my end, I've merged all the "easy" bits, if you could rebase #142 then I could do another review pass on it.

**Pull request closed without merging**